src/utils/payloadStringify.test.js

```diff
@@ -3,6 +3,7 @@ import {
   LUMIGO_SECRET_MASKING_ALL_MAGIC,
   LUMIGO_SECRET_MASKING_EXACT_PATH,
   LUMIGO_SECRET_MASKING_REGEX,
+  OMITTING_KEYS_REGEXES,
```

Comment:
I think we don't need this imported here - any value would have been masked before the fix, and any value would be kept after it, so it's fine to some dummy const value - using this special value confused my a bit as I thought it has a special meaning in this case.

Reply:
Done